### PR TITLE
remove _copy_without_render

### PR DESCRIPTION
**Pull request by dtrifiro**
Not working due to https://github.com/cookiecutter/cookiecutter/issues/1757

### CODE REVIEW

File: cookiecutter.json [hunk 1]
```diff
@@ -20,10 +20,5 @@
         "Development Status :: 5 - Production/Stable",
         "Development Status :: 6 - Mature",
         "Development Status :: 7 - Inactive"
-    ],
-    "_copy_without_render": [
-        ".github/workflows/release.yaml",
-        ".github/workflows/tests.yaml",
-        ".github/workflows/update-template.yaml"
```
Comment (skshetry) [inline cookiecutter.json:27]: The `update-template.yaml` needs escaping:

https://github.com/iterative/py-template/blob/f29fa72d72751294596bb5752c221acdb10f646d/%7B%7Bcookiecutter.project_name%7D%7D/.github/workflows/update-template.yaml#L23

See below on how to escape:
https://github.com/iterative/py-template/blob/f29fa72d72751294596bb5752c221acdb10f646d/%7B%7Bcookiecutter.project_name%7D%7D/.github/workflows/release.yml#L38

Comment (dtrifiro) [reply cookiecutter.json:27]: How about using:
```
 if: {% raw %}${{ steps.update.outputs.changes != '' }}{% endraw %}
```
This makes the intended expression a bit more readable imho

Comment (skshetry) [reply cookiecutter.json:27]: yeah, that's easier to write too. Let's do this. :)

**Review by skshetry — APPROVED** [review]

**Comment by skshetry** [comment]
Thankyou @dtrifiro.

**Pull request merged**